slash/react/src/Form/core/FieldForm.tsx

```diff
@@ -239,7 +239,7 @@ export const renderedChildren = ({
 
 export const FieldForm = ({
   children,
-  message = null,
+  message,
```

Comment:
Pourquoi tu ne met init pas message avec un chaine vide cela éviterez le `safeMessage `non ?

```suggestion
  message = "",
```